jenkinsapi_tests/unittests/test_build_scm_git.py

```diff
@@ -49,3 +49,13 @@ def test_git_revision_branch(build):
     assert build.get_revision_branch()[0]['SHA1'] == \
         '7def9ed6e92580f37d00e4980c36c4d36e68f702'
     assert build.get_revision_branch()[0]['name'] == 'origin/unstable'
+
+
+def test_git_repo_url(build):
+    """
+    Can we extract git repo url for a given build
+    """
+    if not isinstance(build.get_repo_url(), str):
+        raise AssertionError
+    if not build.get_repo_url() == 'https://github.com/salimfadhley/jenkinsapi.git':
+        raise AssertionError
```

Comment:
Why this is better than asserts in function above?

Reply:
I wrote the assert similar to one above , but the py coverage rejected it sayinh not meeting pylint standards.

Reply:
Please revert it to normal pytest assertions. The warning that asserts will be stripped out doesn't apply in this case.

Reply:
done

Reply:
The build failed with checks again when i reverted the changes
Showing 1 file with new issues 
jenkinsapi_tests/unittests/test_build_scm_git.py . Below is the error

Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.
58
assert isinstance(build.get_repo_url(), str)
Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.
59
assert build.get_repo_url() == 'https://github.com/salimfadhley/jenkinsapi.git'
Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.
60
assert len(build.get_repo_url()) == 1
![image](https://user-images.githubusercontent.com/5169211/39886249-c8c5c8b8-5443-11e8-838f-d65d7d8ac09d.png)